src/store/model/bestbuy-ca.ts

```diff
@@ -2,17 +2,13 @@ import {Store} from './store';
 
 export const BestBuyCa: Store = {
 	labels: {
-		inStock: {
-			container: '.addToCartContainer_2uzan',
-			text: ['add to cart']
-		},
```

Comment:
I think we'll want to keep some form of `inStock`. This is good for false positive detection.